roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -97,15 +97,14 @@ policyConfig:
   bootstrapPolicyFile: {{ openshift_master_policy }}
   openshiftInfrastructureNamespace: openshift-infra
   openshiftSharedResourcesNamespace: openshift
-{# TODO: Allow users to override projectConfig items #}
 projectConfig:
-  defaultNodeSelector: "{{ openshift.master.default_node_selector | default("") }}"
-  projectRequestMessage: ""
-  projectRequestTemplate: ""
+  defaultNodeSelector: {{ openshift.master.default_node_selector }}
```

Comment:
The message in particular looked weird without quotes but I deleted the `self-provisioners` role and it showed up on the landing page.

```
projectConfig:
  defaultNodeSelector: type=user-node,zone=east
  projectRequestMessage: Contact your system administrator at projectname@example.com to request a project.
  projectRequestTemplate: default/project-request
  securityAllocator:
    mcsAllocatorRange: s0:/2
    mcsLabelsPerProject: 5
    uidAllocatorRange: 1000000000-1999999999/10000
```